cpp/src/barretenberg/benchmark/honk_bench/honk.bench.cpp

```diff
@@ -29,69 +33,70 @@ void generate_test_plonk_circuit(auto& composer, size_t num_gates)
 /**
  * @brief Benchmark: Creation of a Standard Honk prover
  */
-void create_prover_bench(State& state) noexcept
+void create_prover_standard(State& state) noexcept
 {
     for (auto _ : state) {
         state.PauseTiming();
         auto num_gates = 1 << (size_t)state.range(0);
-        auto composer = proof_system::honk::StandardHonkComposer(static_cast<size_t>(num_gates));
-        generate_test_plonk_circuit(composer, static_cast<size_t>(num_gates));
+        auto composer = Composer(static_cast<size_t>(num_gates));
+        generate_test_circuit(composer, static_cast<size_t>(num_gates));
         state.ResumeTiming();
 
         composer.create_prover();
     }
 }
-BENCHMARK(create_prover_bench)->DenseRange(MIN_LOG_NUM_GATES, MAX_LOG_NUM_GATES, 1)->Repetitions(NUM_REPETITIONS);
+BENCHMARK(create_prover_standard)->DenseRange(MIN_LOG_NUM_GATES, MAX_LOG_NUM_GATES, 1)->Repetitions(NUM_REPETITIONS);
 
 /**
- * @brief Benchmark: Creation of a Standard Honk verifier
+ * @brief Benchmark: Construction of a Standard Honk proof
  */
-void create_verifier_bench(State& state) noexcept
+void construct_proof_standard(State& state) noexcept
 {
+    auto num_gates = 1 << (size_t)state.range(0);
```

Comment:
You can use range multiplier (https://github.com/google/benchmark/blob/main/docs/user_guide.md#custom-counters) as an alternative if you want the exponent to be the metric.

Reply:
Thanks, I feel like there's lots of fun capabilities here that I need to dig into. For now I'll leave it as is since I'm not even sure we care about the standard honk benchmarks much anymore